backend/src/migrations/20230826223636_add_role_to_users_table.ts

```diff
@@ -0,0 +1,23 @@
+import { type Knex } from 'knex';
```

Comment:
Name migrations like:

`add_${tableName}_table.ts` - to create table
`update_${tableName}_table.ts` - to update table